test/parallel/test-fs-writev.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+tmpdir.refresh();
+
+const expected = 'ümlaut. Лорем 運務ホソモ指及 आपको करने विकास 紙読決多密所 أضف';
+
+const getFileName = (i) => path.join(tmpdir.path, `writev_${i}.txt`);
+
+/**
+ * Testing with a array of buffers input
+ */
+
+// fs.writev with array of buffers with all parameters
+{
+  const filename = getFileName(1);
+  fs.open(filename, 'w', 0o644, common.mustCall((err, fd) => {
```

Comment:
Minor nit: since we're already using `fs.closeSync()` we could save a level of indentation and simply use `fs.openSync()` instead.